### PR TITLE
Removes blob from rotation

### PR DESCRIPTION
**Pull request by ghost**
:cl: Removed blob
del: Removed blob.
/:cl:

Although I enjoy blob, the playerbase doesn't.

### CODE REVIEW

**Comment by tortellinitony** [comment]
do a vote

**Comment by PopNotes** [comment]
One part of the playerbase speaking, I like blob.

**Comment by tortellinitony** [comment]
yeah me too

**Comment by gumner** [comment]
no gtfo

File: config/game_options.txt [hunk 1]
```diff
@@ -87,7 +87,7 @@ PROBABILITY CULT 2
 PROBABILITY CHANGELING 2
 PROBABILITY WIZARD 4
 PROBABILITY MALFUNCTION 2
-PROBABILITY BLOB 2
+PROBABILITY BLOB 0
```
Comment (gumner) [inline config/game_options.txt:90]: wohah you acidentally typed a 0 instead of 3

**Comment by tortellinitony** [comment]
too bad this file isn't the server's actual config hahahahha

**Comment by j-mie** [comment]
Do a server vote.

**Pull request closed without merging**